bots/cloudtrail_enable_log_file_validation.py

```diff
@@ -0,0 +1,31 @@
+"""
+## cloudtrail_enable_log_file_validation
+What it does: Enable log file validation in cloudTrail
+Usage: AUTO: cloudtrail_enable_log_file_validation.py
+Limitations: None
+Defaults:
+"""
+from botocore.exceptions import ClientError
+
+
+def run_action(boto_session, rule, entity, params):
+    # create a cloudTrail session
+    cloudTrail_client = boto_session.client('cloudtrail')
+
+    cloudTrail_name = entity['name']
```

Comment:
Can you use entity id instead of name?
Maybe first get all then filter by id?

Reply:
change it to entity['id']